docs/index.md

```diff
@@ -45,206 +45,83 @@ theme:
-pip install "napari[all]"
-```
-````
-
-### From conda
-
-```sh
-conda install -c conda-forge napari
-```
-
-### Current development branch from github
-
-To install the current `main` branch on github (which will usually be ahead
-of the latest release on pypi)
-
-```sh
-pip install "git+https://github.com/napari/napari.git#egg=napari[all]"
-```
-
-For more information or troubleshooting see our [installation
-tutorial](./tutorials/fundamentals/installation)
-
-````{admonition} Specifying a GUI Backend
-:class: tip
-
-napari needs a library called [Qt](https://www.qt.io/) to run its user interface
-(UI). In Python, there are two alternative libraries to run this, called
-[PyQt5](https://www.riverbankcomputing.com/software/pyqt/download5) and
-[PySide2](https://doc.qt.io/qtforpython/). By default, we don't choose for you,
-and simply running `pip install napari` will not install either. You *might*
-already have one of them installed in your environment, thanks to other
-scientific packages such as Spyder or matplotlib. If neither is available,
-running napari will result in an error message asking you to install one of
-them.
-
-Running `pip install "napari[all]"` will install the default framework – currently
-PyQt5, but this could change in the future.
-
-To install napari with a specific framework, you can use:
-
-```sh
-pip install "napari[pyqt5]"    # for PyQt5
-
-# OR
-pip install "napari[pyside2]"  # for PySide2
-```
-````
-
-## Getting started
-
- If you are new to napari, check out our
- [getting started guide](tutorials/fundamentals/getting_started.md).
-
-### Example
-
-(The example below requires the `scikit-image` package to run. We just use data
-samples from this package for demonstration purposes. If you change the example
-to use your own dataset, you may not need to install this package.)
-
-From inside an IPython shell or jupyter notebook you can open up an interactive
-viewer by calling:
+Napari is a fast, multi-dimensional image viewer for the Python programming
+language. It also aims to provide GUI-based access to the scientific Python
+image analysis ecosystem.
 
 ```python
-from skimage import data
 import napari
-
-viewer = napari.view_image(data.astronaut(), rgb=True)
+viewer = napari.view_image(my_array)
 ```
 
-![image](./images/screenshot-add-image.png)
+![image: napari viewer with cryoET data](images/alisterburt-viewer-cryoet.webm)
 
-To do the same thing inside a script call (note the call to {func}`napari.run` at
-the end)
+Napari can help you:
 
-```python
-from skimage import data
-import napari
+- quickly *view* 2D, 3D, and higher-dimensional image data
+- *overlay* segmentations (labels), points, vectors, polygons, surface meshes,
+  vectors, and tracking data
+- *annotate* data, seamlessly weaving human curation with scripts and
+  interactive sessions using Python
+- *analyze* data with a [rich ecosystem of plugins](https://napari-hub.org)
+- and [more!](https://napari.org/community)
```

Comment:
@sofroniewn fair point, though it wasn't quite a typo — I guess I meant to succinctly say, "if napari doesn't meet your needs, get in touch! Maybe it can!"

Reply:
ah well - fine if you want to change it to say that! i just read it more of "find out what napari can do here" - which I think i'd probably point people to first